inst/sympref.m

```diff
@@ -428,7 +431,8 @@
 %!test
 %! syms x
 %! r = sympref('reset');
+%! % restore original sympref settings
+%! sympref ('ipc',   sympref_orig.ipc);
+%! sympref ('quiet', sympref_orig.quiet);
 %! syms x
 %! assert(r)
-%! % ok, can be noisy again
-%! sympref('quiet', 'default')
```

Comment:
@cbm755 Not sure about this, was the purpose of this last test block to simply restore default state? Or am I screwing something up, should I add another block after this one to reset yet again and restore the settings?